docs/operators-and-hooks-ref.rst

```diff
@@ -147,9 +147,91 @@ Software operators and hooks
 These integrations allow you to perform various operations within software developed by Apache Software
 Foundation.
 
-.. operators-hooks-ref::
-   :tags: apache
-   :header-separator: "
+.. list-table::
```

Comment:
This table is generated automatically. You don't have to change it. Can you undo this change?

Reply:
so delete lines 150 - 235?